src/Apps/Feature/Components/FeatureFeaturedLink/FeatureFeaturedLink.tsx

```diff
@@ -0,0 +1,91 @@
+import React from "react"
+import styled from "styled-components"
+import { space, SpaceProps } from "styled-system"
+import { Box, Sans, Serif, Image, Flex } from "@artsy/palette"
+
+const Container = styled.a`
+  ${space}
+  display: flex;
+  text-decoration: none;
+  flex-direction: column;
+`
+
+const Figure = styled(Box)`
+  position: relative;
+`
+
+const Cover = styled(Image)`
+  display: block;
+  width: 100%;
+  height: 100%;
+`
+
+const Title = styled(Sans)`
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  background: linear-gradient(
+    rgba(255, 255, 255, 0) 0%,
+    rgba(0, 0, 0, 0.25) 100%
+  );
+`
+
+const Meta = styled(Flex).attrs({ mt: 1 })``
+
+const MetaCell = styled(Flex)`
+  flex-basis: 50%;
+`
+
+export type FeatureFeaturedLinkProps = SpaceProps & {
+  wide?: boolean
+  featuredLink: {
+    title: string
+    href: string
+    subtitle?: string
+    description?: string
+    image: {
+      width: number
+      height: number
+      url: string
+    }
+  }
+}
+
+export const FeatureFeaturedLink: React.FC<FeatureFeaturedLinkProps> = ({
+  wide,
+  featuredLink: { title, href, subtitle, description, image },
+  ...rest
+}) => {
+  return (
+    <Container href={href} {...rest}>
+      {image ? (
+        <Figure>
+          <Cover src={image.url} />
```

Comment:
Will be a ResponsiveImage but I'd be interested in fixing that up a bit.

Reply:
Need real data for placeholders blah blah.

Reply:
I need to look at the Gemini templates for FeaturedLink images but I believe they all crop to a 4:3 rectangle. cc @sweir27  — noticed the designs are specified around portrait aspect ratios.

Reply:
Oh really?? Ideally we'd pull the version that has the original aspect ratio and then we can pass it through the `crop` endpoint (using that helper here or via the MP call) to get the size we want.

The only caveat with that is we should make sure to do it based on fixed sizes (even via breakpoint) vs. dynamic based on exact browser size. This is just bc we'll want to benefit from caching which will break down if the request is highly variable.

Reply:
I don't think this template has a normalized version unfortunately, we'll probably have to add it and then reprocess. But I'll double check the template.

https://github.com/artsy/quartz/blob/master/src/gravity/FeaturedLink.ts#L23-L29

Reply:
OK so it looks like we can use the `wide` version (tho it's a bit weird), the rest of them are all fill crops to 4:3 or 1:1.
```
 height: 480
 width: 640
 quality: 90
 resize_to: "fill"
 key: "large_rectangle"

 height: nil
 width: 2000
 quality: 85
 resize_to: "width"
 key: "wide"

 height: 400
 width: 400
 quality: 90
 resize_to: "fill"
 key: "large_square"

 height: 200
 width: 200
 quality: 90
 resize_to: "fill"
 key: "medium_square"

 height: 100
 width: 100
 quality: 90
 resize_to: "fill"
 key: "small_square"

 height: 195
 width: 260
 quality: 90
 resize_to: "fill"
 key: "medium_rectangle"

 height: 113
 width: 148
 quality: 90
 resize_to: "fill"
 key: "small_rectangle"
```